cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -387,7 +391,7 @@ code <<-EOS
       },
     0)
 
-    var final_average = parseFloat(sum/averages.length)
+    var final_average = parseFloat(sum/averages.length).toFixed(2)
```

Comment:
.toFixed(2) will convert the number to a string, which will adversely affect sorting. Instead, encapsulate the toFixed inside the parseFloat, like so:

var final_average = parseFloat((sum / averages.length).toFixed(2))

Reply:
Done, Thank you